check_diff/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use clap::Parser;
+/// Inputs for the check_diff script
+#[derive(Parser)]
+struct CliInputs {
+    /// Git url of a rustfmt fork to compare against the latest master rustfmt
+    remote_repo_url: String,
+    /// Name of the feature branch on the forked repo
+    feature_branch: String,
+    /// Optional commit hash from the feature branch
+    #[arg(short, long)]
+    commit_hash: Option<String>,
+    /// Optional comma separated list of rustfmt config options to pass when running the feature branch
+    #[clap(value_delimiter = ',', short, long,num_args = 1..)]
+    rustfmt_config: Option<Vec<String>>,
```

Comment:
Can you explain why you're using `#[arg()]` for `commit_hash` and using `#[clap()]` for `rustfmt_config`? At first glance it seems a little inconsistent. The [clap v4.4.2 derive  reference](https://docs.rs/clap/4.4.2/clap/_derive/index.html) uses `#[arg()]` so my first thought is to do the same. Also, maybe `#[clap()]` is deprecated since it was removed from the codebase in this PR: https://github.com/rust-lang/rustfmt/pull/6101